EMCAL/EMCALbase/AliEMCALSimParam.cxx

```diff
@@ -131,5 +134,9 @@ void AliEMCALSimParam::Print(Option_t *) const
   printf("\t sdigitization parameters       A = %f\n",     fA);
   printf("\t                                B = %f\n",     fB);
   printf("\t Threshold for EC Primary assignment  = %f\n", fECPrimThreshold);
+
+  printf("=== Parameters in TriggerElectronics === \n");
+  printf("\t L1 FastOR timesum noise (fL1ADCNoise) = %f\n", fL1ADCNoise);
+
```

Comment:
Please rather implement operator<< for AliEMCALSimParams - we are in the c++ time and the time for such print functions are over. We usually do the following:

- Add PrintStream functions (in header: include iosfwd instead of iostream) with the following interface
``` c++
void PrintStream(std::ostream &stream) const;
```
  and implement the printing in the same way as if one would use std::cout.
- Implement operator<< becomes now trivial:
``` c++
std::ostream &operator<<(std::ostream &stream, const AliEMCALSimParam &param) {   
  param.PrintStream(stream);
  return stream;
}
```
- Finally the print function becomes:
``` c++ 
void AliEMCALSimParam::Print(Option_t *) {
  std::cout << *this << std::endl;
}
```

This adds more flexibility to the debug streaming as besides the print function this can be used in std::cout, but better with any kind of logging library like the loggers in AliLog (AliErrorStream, AliWarningStream, AliInfoStream, AliDebugStream) ... where log levels can be set

Reply:
For the Print(Option_t *) function, should it be AliInfoStream() or AliDebugStream() instead of std::cout?

Reply:
This is basically a compatibility function to ROOT5 style logging where I think std::cout is the way to go.